source/braille.py

```diff
@@ -1925,6 +1926,25 @@ def getPossiblePorts(cls):
 	#: @type: L{inputCore.GlobalGestureMap}
 	gestureMap = None
 
+	@classmethod
+	def _getModifierGestures(cls):
+		"""Retrieves modifier gestures from this display driver's L{gestureMap}
+		that are bound to modifier only keyboard emulate scripts.
+		@return: the ids of the display keys and the associated generalised modifier names
+		@rtype: generator of (set, set)

```

Comment:
Why sets instead of tuples/lists? Sets have some fancy operator overloads, but that pattern is not used much in NVDA.

Reply:
I use sets because of these operator overloads.